src/escpos/printer/cups.py

```diff
@@ -20,6 +20,9 @@
     import cups
 
     _DEP_PYCUPS = True
+    # Store server defaults before further configuration
+    DEFAULT_HOST = cups.getServer()
```

Comment:
Yes, it is safe.
`cups.getServer()` and `cups.getPort()`have internal defaults so a value is always returned if the `pycups` library is successfully imported.

In fact, I've successfully disabled the cups service on my machine and then import and instance a `CupsPrinter()` object .
Of course it will raise an exception after you send a print command but that's the expected behavior.

Reply:
Ok, great. Thank you for the details!